genref/config.yaml

```diff
@@ -179,3 +179,9 @@ apis:
     title: kubeadm Configuration (v1beta2)
     package: github.com/tengqm/kubeconfig
     path: config/kubeadm/v1beta2
+
+  # This is for testing
```

Comment:
Is this used for testing?

Reply:
Yes. The package is deeply embedded in k/k repo. To generate reference docs for kubeadm, we have to clone the relevant files somewhere (currently under github.com/tengqm/kubeconfig as you can see from line 180).